### DISCUSSION TITLE
Option 1: Getting The Image

### DISCUSSION THREAD

**Discussion by j6s33m** [moralmunky/Home-Assistant-Mail-And-Packages#417] [Q&A] [answered]
My "sensor.mail_image_url" is pulling the correct url when I double click into the entity, but the gif won't show up in my lovelace dashboard using the custom Mail & Packages card. The only way it will show up is if I hard code the url...the sensor url won't pull it in.

<img width="444" alt="Screen Shot 2021-03-26 at 1 01 10 PM" src="https://user-images.githubusercontent.com/17929290/112673965-5a09cc80-8e33-11eb-8938-4d40f806e927.png">

@moralmunky

**Comment by firstof9** [comment, marked as answer]
I think you need to be running the pre-release/beta version of the card for that to work.

**Reply by j6s33m** [reply]
no dice. just updated to beta version and am still seeing blank blank image.

**Reply by j6s33m** [reply]
I switched to the camera entity and that seems to be working now.

**Reply by moralmunky** [reply]
I don't see a version number after the last checked time. The 0.6 of the card took the sensor as input. Pre 0.06 you supplied the url yourself. 0.06 is a prerelease. The card is being rewritten for the next official release of the integration.
